### PR TITLE
Users patch branch 0.6

### PR DESCRIPTION
**Pull request by DiegoRuiz454**
# **Description**

PATCH api/users/{id} query parameter implemented. users.controllers.js and users.router.js. the user can edit and update: full_name, position, linkedin,github, website, profile_image_url. What if the id does not exists? the following message will appear: `the cv with the id number :'${req.params.id}' doesn't exist in our database. Please check again!`

Fixes # (29)

# **How to test?**

type in the terminal npm run dev. after test it with POSTMAN select PATCH while using the endpoint
http://localhost:5000/api/users/{id} after that press send. 

# **Checklist**

- [☑] I have performed a self-review of my own code
- [☑] I have commented my code, particularly in hard-to-understand areas
- [☑] I have made corresponding changes to the documentation
- [☑] This PR is ready to be merged and not breaking any other functionality
users 1
users does not exist

### CODE REVIEW

**Comment by pdsorensen** [comment]
Hi @DiegoRuiz454 , it looks good! Can i have you look at files changed, and go each of the github warning/errors related to your files? 

Also, i don't understand why so many deletions is happening in the swagger file. I think this is unintended? :)

File: src/server/api/routes/users.router.js [hunk 1]
```diff
@@ -10,138 +7,44 @@ const usersController = require('../controllers/users.controller');
...
-    .then((result) => res.json(result))
-    .catch(next);
-});
-
-/**
- * @swagger
- * /modules/{ID}:
- *  get:
- *    summary: Get user by ID
+ * /cvs/{ID}:
```
Comment (bytewiz) [inline src/server/api/routes/users.router.js:10]:
```suggestion
 * /users/{ID}:
```

File: src/server/api/routes/users.router.js [hunk 2]
```diff
@@ -10,138 +7,44 @@ const usersController = require('../controllers/users.controller');
...
-});
-
-/**
- * @swagger
- * /modules:
- *  post:
- *    summary: Create a user
- *    description:
- *      Will create a user.
+ *      Will create a new user.
```
Comment (bytewiz) [inline src/server/api/routes/users.router.js:14]: Will it create a new user or update the user?

**Review by bytewiz — CHANGES_REQUESTED** [review]
You renamed the controller: `users.controller.js` to  `users.controllers.js`
Please avoid that so we keep it consistent and avoid conflicts it should be named.

`users.controller.js`

**Pull request closed without merging**